src/InstagramClone.Api/Controllers/UsersController.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using InstagramClone.Api.Database;
+using InstagramClone.Api.Entities;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace InstagramClone.Api.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class UsersController : ControllerBase
+    {
+        private readonly InstagramCloneDbContext _context;
+        public UsersController(InstagramCloneDbContext context)
+        {
+            _context = context;
+        }
+        
+        [HttpGet("{Id}")]
+        public async Task<ActionResult<User>> GetUser(Guid id)
+        {
```

Comment:
can you please research the appropriate return type(s) for web API controller methods and update your action method to use it here.

read https://docs.microsoft.com/en-us/aspnet/web-api/overview/getting-started-with-aspnet-web-api/action-results#ihttpactionresult

...and consider returning an `IHttpActionResult` result

Reply:
Ok, Done